src/components/product_card/ProductCard.jsx

```diff
@@ -42,7 +43,7 @@ const ProductCard = ({ card_type, product }) => {
 					<span className="text-badge ecommerce-chip-left"> Trending </span>
 					<img
 						className="card-img-container img-responsive"
-						src="https://assets.myntassets.com/f_webp,dpr_1.5,q_60,w_210,c_limit,fl_progressive/assets/images/16859196/2022/1/18/bcc4d322-6227-4601-a5f4-44dee3aafd551642484853447ALDOMenWhiteColourblockedSneakers1.jpg"
+						src={image_src}
```

Comment:
In line 63 of this file, you added onClick on the icon instead of on the button that is wrapping it. If the user clicks on the button but not on the icon, the item is not getting added to wishlist.

Reply:
moved onClick to button instead of icon in  new commit.